CHANGELOG.rst

```diff
@@ -14,6 +14,10 @@ Change Log
 Unreleased
 ----------
 
+1.4.5 – 2024-07-25
```

Comment:
nit: could be considered a `feat` release, with a minor version bump to match the semantic commit type